doc/manual/metaprogramming.rst

```diff
@@ -45,19 +45,29 @@ each string into an object called an expression, represented by the Julia type
 
     julia> typeof(ex1)
     Expr
+    
 
 :obj:`Expr` objects contain three parts:
 
-- a ``Symbol`` identifying the kind of expression. A symbol is an
-  `interned string <https://en.wikipedia.org/wiki/String_interning>`_
+.. doctest::
+
+    julia> fieldnames(Expr)
+    3-element Array{Symbol,1}:
+     :head
+     :args
+     :typ 
+
+- `head` -- a ``Symbol`` identifying the 
+  `kind of expression <http://docs.julialang.org/en/release-0.5/devdocs/ast/#expr-types>`_. 
```

Comment:
This link should probably not be a URL but rather use an RST reference to that section.